bitnami/envoy-gateway/templates/certgen/job.yaml

```diff
@@ -0,0 +1,153 @@
+          securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" .Values.certgen.containerSecurityContext "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.certgen.resources }}
+          resources: {{ toYaml .Values.certgen.resources | nindent 12 }}
+          {{- else if ne .Values.certgen.resourcesPreset "none" }}
+          resources: {{- include "common.resources.preset" (dict "type" .Values.certgen.resourcesPreset) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.certgen.customStartupProbe }}
+          startupProbe: {{- include "common.tplvalues.render" (dict "value" .Values.certgen.customStartupProbe "context" $) | nindent 12 }}
+          {{- else if .Values.certgen.startupProbe.enabled }}
+          startupProbe:
+            exec:
+              command:
+                - pgrep
+                - certgen
+            initialDelaySeconds: {{ .Values.certgen.startupProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.certgen.startupProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.certgen.startupProbe.timeoutSeconds }}
+            successThreshold: {{ .Values.certgen.startupProbe.successThreshold }}
+            failureThreshold: {{ .Values.certgen.startupProbe.failureThreshold }}
+          {{- end }}
+          {{- if .Values.certgen.customLivenessProbe }}
+          livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.certgen.customLivenessProbe "context" $) | nindent 12 }}
+          {{- else if .Values.certgen.livenessProbe.enabled }}
+          livenessProbe:
+            exec:
+              command:
+                - pgrep
+                - certgen
+            initialDelaySeconds: {{ .Values.certgen.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.certgen.livenessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.certgen.livenessProbe.timeoutSeconds }}
+            successThreshold: {{ .Values.certgen.livenessProbe.successThreshold }}
+            failureThreshold: {{ .Values.certgen.livenessProbe.failureThreshold }}
+          {{- end }}
+          {{- if .Values.certgen.customReadinessProbe }}
+          readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.certgen.customReadinessProbe "context" $) | nindent 12 }}
+          {{- else if .Values.certgen.readinessProbe.enabled }}
+          readinessProbe:
+            exec:
+              command:
+                - pgrep
+                - certgen
+            initialDelaySeconds: {{ .Values.certgen.readinessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.certgen.readinessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.certgen.readinessProbe.timeoutSeconds }}
+            successThreshold: {{ .Values.certgen.readinessProbe.successThreshold }}
+            failureThreshold: {{ .Values.certgen.readinessProbe.failureThreshold }}
+          {{- end }}
```

Comment:
Do we require probes for a K8s Job?

Reply:
It is expected by some of the kubescape checks